README.md

```diff
@@ -4,7 +4,8 @@ Start and join voice calls, video calls and use screen sharing with your team me
 
 Clicking a video icon in a Mattermost channel posts a message that invites team members to join a Jitsi meetings call.
 
-Originally developed by [Sean Sackowitz](https://github.com/seansackowitz).
+**Maintainer:** [Jesús Espino](https://github.com/jespino)
```

Comment:
Other plugin put this information directly under the title e.g. https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md. 1/5 that this repo should be consistent.